capa/features/extractors/viv/indirect_calls.py

```diff
@@ -20,12 +21,15 @@
 i386ImmOper = envi.archs.i386.disasm.i386ImmOper
 i386ImmMemOper = envi.archs.i386.disasm.i386ImmMemOper
 Amd64RipRelOper = envi.archs.amd64.disasm.Amd64RipRelOper
+ARMRegOper = envi.archs.arm.disasm.ArmRegOper
+ARMImmOper = envi.archs.arm.disasm.ArmImmOper
+ARMScaledOffsetOper = envi.archs.arm.disasm.ArmScaledOffsetOper
 LOC_OP = vivisect.const.LOC_OP
 IF_NOFALL = envi.IF_NOFALL
 REF_CODE = vivisect.const.REF_CODE
 FAR_BRANCH_MASK = envi.BR_PROC | envi.BR_DEREF | envi.BR_ARCH
 
-DESTRUCTIVE_MNEMONICS = ("mov", "lea", "pop", "xor")
+DESTRUCTIVE_MNEMONICS = ("mov", "lea", "ldr", "pop", "xor", "eor")
```

Comment:
```suggestion
DESTRUCTIVE_MNEMONICS = (
    # x86
    "mov", "lea", "pop", "xor",
    # arm
    "ldr", "eor")
```

Reply:
see comment above on mixing archs, something like this could help

Reply:
Yes, great idea, I'll try to apply this where I can. I think it is only necessary for mnemonics and not for constants with arm in their names but it's up to you.

Reply:
even further, we could use sets like:


```py
DESTRUCTIVE_MNEMONICS_X86 = {
    "mov", "lea", "pop", "xor",
}
DESTRUCTIVE_MNEMONICS_ARM = {
    "ldr", "eor",
}
DESTRUCTIVE_MNEMONICS = DESTRUCTIVE_MNEMONICS_X86 | DESTRUCTIVE_MNEMONICS_ARM
```